orchestra/static/orchestra/todos/todo-list.html

```diff
@@ -100,6 +101,7 @@ <h3>
               skip-todo="todoList.skipTodo"
               unskip-todo="todoList.unskipTodo"
               task-slugs="todoList.taskSlugs"
+              on-toggle-todo="todoList.onToggleTodo"
```

Comment:
the rest of them  aren't preceded wtih "on-" :)

Reply:
The rest of them are actually taking actions like "skip" or "unskip" whereas `on-toggle-todo` is an event handler. It does not toggle the todo. It is handling the event when a todo is toggled.